lib/collection/proxy-config.js

```diff
@@ -85,9 +116,22 @@ _.assign(ProxyConfig.prototype, /** @lends ProxyConfig.prototype */ {
         if (!_.isObject(options)) {
             return;
         }
-        _.has(options, 'server') && this.server.update(options.server);
-        _.isString(options.match) && (this.match = new UrlMatchPattern(options.match));
-        _.has(options, 'server') && (this.tunnel = _.isBoolean(options.tunnel) ? options.tunnel : false);
+        _.isString(options.host) && (this.host = options.host.replace(regexes.protocolMatcher, ''));
+        _.isInteger(options.port) && (options.port >= MIN_PORT && options.port <= MAX_PORT) && (this.port = options.port);
```

Comment:
Do not add port min max logic here. As a rest client, we should allow all port values.